src/features/DisplayControls/NursingTasks/components/UpdateNursingTasks.jsx

```diff
@@ -56,7 +60,10 @@ const UpdateNursingTasks = (props) => {
 
   const { config } = useContext(IPDContext);
   const { nursingTasks = {} } = config;
-
+  const { drugChart = {} } = config;
```

Comment:
we can define both of them in a single line like this:
const { nursingTasks = {},  drugChart = {} } = config;